src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php

```diff
@@ -494,6 +500,20 @@ private function handleImportThemeException(Exception $e)
                     '%theme_name%' => $e instanceof ImportedThemeAlreadyExistsException ? $e->getThemeName()->getValue() : '',
                 ]
             ),
+            ThemeConstraintException::class => [
+                ThemeConstraintException::RESTRICTED_ONLY_FOR_SINGLE_SHOP => $couldntToUploadTheme . ': ' . $this->trans(
+                        'Themes can be changed only in single shop context', 'Admin.Notifications.Error'
+                    ),
+                ThemeConstraintException::MISSING_CONFIGURATION_FILE => $couldntToUploadTheme . ': ' . $this->trans(
+                        'Missing configuration file', 'Admin.Notifications.Error'
+                    ),
+                ThemeConstraintException::INVALID_CONFIGURATION => $couldntToUploadTheme . ': ' . $this->trans(
+                        'Invalid configuration', 'Admin.Notifications.Error'
+                    ),
+                ThemeConstraintException::INVALID_DATA => $couldntToUploadTheme . ': ' . $this->trans(
+                        'Invalid data', 'Admin.Notifications.Error'
+                    ),
+            ],
```

Comment:
Better to use `sprintf`

Reply:
Hey removed first part of message in general so sprintf is no longer needed.